providers/gcloud/gcloudProvider.go

```diff
@@ -288,21 +290,27 @@ func (g *gcloudProvider) GetDomainCorrections(dc *models.DomainConfig) ([]*model
 	}
 
 	chg := &gdns.Change{Kind: "dns#change"}
+	const maxGroup = 2000
```

Comment:
Is this based on a limit set by the gcloud API?

Reply:
No, I was just guessing numbers.